examples/rust/extend_viewer_ui/Cargo.toml

```diff
@@ -0,0 +1,28 @@
+[package]
+name = "extend_viewer_ui"
+version = "0.6.0"
+edition = "2021"
+rust-version = "1.69"
+license = "MIT OR Apache-2.0"
+publish = false
+
+[features]
+default = []
+
+# Turn on if you want to share analytics with Rerun (e.g. callstacks on crashes).
+analytics = ["re_crash_handler/analytics", "re_viewer/analytics"]
```

Comment:
I reckon this is opt-in since we want to force the user to make a conscious choice to have analytics that may include their own callstack? A bit unfortunate since this might make us miss a lot of analytics going forward 🤔

Reply:
Yeah, exactly. I'm not sure if we want to see a lot of analytics from `unwrap()`s in our users code though.